code/modules/mapping/space_management/traits.dm

```diff
@@ -49,6 +49,15 @@
 				. += S.z_value
 				break
 
+/// Get a list of all z which have all of the specified traits
+/datum/controller/subsystem/mapping/proc/levels_by_all_traits(z, list/traits)
+	. = list()
+	var/list/_z_list = z_list
```

Comment:
```suggestion
	var/list/z_list = src.z_list
```
you could do this, or just rename it. There is a style guide entry for this but it explicitly says in /New or /Initialize

Reply:
Actually, whats even the point? You don't modify this list at all